lighthouse-viewer/test/test-helpers.js

```diff
@@ -14,8 +14,9 @@ const jsdom = require('jsdom');
 const PAGE = fs.readFileSync(path.join(__dirname, '../app/index.html'), 'utf8');
 
 function setupJsDomGlobals() {
-  global.document = jsdom.jsdom(PAGE);
-  global.window = global.document.defaultView;
+  const {window} = new jsdom.JSDOM(PAGE);
```

Comment:
this is throwing an error in the test because viewer has its own version in its `package.json`. I'd vote for removing it there so it can just use the root directory's version